src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/OnlineHearing.java

```diff
@@ -0,0 +1,139 @@
+package uk.gov.hmcts.reform.sscs.domain.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.Objects;
+
+@JsonInclude(value = JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OnlineHearing {
```

Comment:
Think this can be removed

Reply:
OnlineHearing data object is used by both MYA and COR, can't removed it unless if we modify the frontend